core/ir/opnd_shared.c

```diff
@@ -1785,6 +1804,7 @@ opnd_size_in_bytes(opnd_size_t size)
     case OPSZ_VAR_REGLIST: return 0; /* varies to match reglist operand */
     case OPSZ_xsave:
         return 0; /* > 512 bytes: client to use drutil_opnd_mem_size_in_bytes */
+    case OPSZ_CACHE_LINE: return (uint)get_dcache_zva_size();
```

Comment:
We're tying "cache line size" with "dcache zva size" here. In other places, we're keeping variables for these separate because they can potentially be different. Perhaps we should change `OPSZ_CACHE_LINE` with `OPSZ_DCACHE_ZVA`?